src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -475,14 +482,9 @@ class InferenceProfiler {
 
   /// Helper function to perform measurement.
   /// \param status_summary The summary of this measurement.
-  /// \param measurement_window Indicating the number of requests or the
-  /// duration in milliseconds to collect requests.
-  /// \param is_count_based determines whether measurement_window is indicating
-  /// time or count.
+  /// \param config The configuration for measurement
```

Comment:
```suggestion
  /// \param config The configuration for measurement.
```

Nit: Adding period for consistency.